docker/build_dagmc.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
```

Comment:
Optional: For dockerfiles, I often also set `-x` so it will print the commands as it runs [1].  This is helpful in debugging bash files when it fails.  You can set both as follows:

```
set -ex
```

[1] https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin